src/main/java/com/parasoft/findings/sonar/importer/ParasoftIssuesParser.java

```diff
@@ -113,16 +113,21 @@ public int loadFindings(File file)
         return loadedFindings;
     }
 
-    public Set<IRuleViolation> getFindings(String inputFilePath)
+    public Set<IRuleViolation> getFindings(File inputFile)
     {
-        return _violations.get(inputFilePath);
+        return _violations.get(inputFile);
     }
 
     public int createNewIssues(InputFile sourceFile, ParasoftProduct product, SensorContext context)
     {
         ActiveRules activeRules = context.activeRules();
-        String inputFilePath = sourceFile.uri().getPath();
-        var findings = getFindings(inputFilePath);
+        File source;
+        if (sourceFile.isFile()) {
+            source = new File(sourceFile.uri().getPath()).getAbsoluteFile();
+        } else {
+            return 0;
+        }
+        var findings = getFindings(source);
         if (CollectionUtil.isEmpty(findings)) {
             Logger.getLogger().info(NLS.getFormatted(Messages.NoFindingsFor, sourceFile.toString()));
```

Comment:
There are two kinds of path info in log, I think we may want to use absolute path for all paths, it will help us to debug when no violations created in sonar.
![image](https://github.com/parasoft/parasoft-findings-sonar/assets/109122571/92e0e869-f1b0-4e0e-852f-ab2a992c6604). @yshi-parasoft How do you thinks about it?

Reply:
I agree with you. It's better to be consistent, and an absolute path is more useful than a relative path. 
This looks like an existing behavior, right? You can create a new pull request or, if it's not much work, you can check it into this pull request.

Reply:
The logs looks like this now:
![image](https://github.com/parasoft/parasoft-findings-sonar/assets/109122853/a627c8b4-3e2c-4daf-89cd-c39f25b635f8)